docs/docs/topics/tcp-support.md

```diff
@@ -0,0 +1,103 @@
+---
+title: TCP Support
+description: >-
+  This article describes how to leverage pomerium for TCP proxying
+---
+
+# TCP Support
+
+## About
+
+Internal applications are often HTTP, but the engineering teams that support those applications frequently require access to lower level administrative and data protcols such as SSH, RDP, Postgres, MySQL, Redis, etc.
+
+Though Pomerium is primarily designed to authorize and proxy HTTP traffic, it can also be leveraged to protect these non-HTTP systems with the same consistent authorization policy.  This is achieved by tunneling TCP over HTTP with the help of a client side command built into [`pomerium-cli`](/docs/installation.md#pomerium-cli).
+
+
+Internally, Pomerium uses the [`CONNECT` method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/CONNECT) to establish the TCP tunnel.
+
+::: warning
+For proper TCP support, you should not use an HTTP proxy in front of Pomerium.  Instead, configure your load balancer in L4 or TCP mode.
+
+If an HTTP proxy is in front of Pomerium, it must know how to properly handle the `CONNECT` command and proxy it upstream.  This will be specific to each proxy implementation.
```

Comment:
```suggestion
Otherwise, the HTTP proxy fronting Pomerium must know how to properly handle the `CONNECT` command and proxy it upstream.  This will be specific to each proxy implementation.
```